bowling_game.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+# Esta clase es para definir el juego
+class Bowling
+  def initialize
+    @total_pins = 0
+    @frames = []
+    @current_frame = Frame.new
+  end
+
+  def roll(pins_down)
+    @current_frame << pins_down
+
+    (unless (@current_frame.done? && @frames.length <= 9) || (@frames.length == 10 &&
+      (@frames.last.spare? || @frames.last.strike?))
+      return end)
+    @frames << @current_frame
```

Comment:
I don't know why you're still refusing to use `attr_accessor attr_reader attr_writer` etc. :sweat_smile: